css/cmubuggy-bootstrap.css

```diff
@@ -1875,7 +1696,7 @@ pre code {
   box-shadow: 0 0 0 0.2rem rgba(226, 35, 53, 0.25);
 }
 
-.form-control::-webkit-input-placeholder {
+.form-control::-moz-placeholder {
```

Comment:
Do we not still need the webkit version?

Reply:
The changes that SASS made to this file are a bit beyond my CSS understanding since they mostly play with browser versions.

However, if I were to guess, it would be because all currently supported versions of Safari and Chrome currently use "placeholder" directly and don't need webkit-placeholder any more (and there is an identical block a few lines down for that selector).